Kernel/Arch/i386/CPU.cpp

```diff
@@ -71,56 +71,56 @@ extern "C" void pre_init_finished(void);
+        "    pushl %ds\n"                                                                                                                                                    \
+        "    pushl %es\n"                                                                                                                                                    \
+        "    pushl %fs\n"                                                                                                                                                    \
+        "    pushl %gs\n"                                                                                                                                                    \
+        "    pushl %ss\n"                                                                                                                                                    \
+        "    mov $" __STRINGIFY(GDT_SELECTOR_DATA0) ", %ax\n"                                                                                                                \
+                                                    "    mov %ax, %ds\n"                                                                                                     \
+                                                    "    mov %ax, %es\n"                                                                                                     \
+                                                    "    mov $" __STRINGIFY(GDT_SELECTOR_PROC) ", %ax\n"                                                                     \
+                                                                                               "    mov %ax, %fs\n"                                                          \
+                                                                                               "    pushl %esp \n" /* set TrapFrame::regs */                                 \
+                                                                                               "    subl $" __STRINGIFY(TRAP_FRAME_SIZE - 4) ", %esp \n"                     \
+                                                                                                                                             "    pushl %esp \n"             \
+                                                                                                                                             "    cld\n"                     \
+                                                                                                                                             "    call enter_trap_no_irq \n" \
+                                                                                                                                             "    call " #title "_handler\n" \
+                                                                                                                                             "    jmp common_trap_exit \n");
```

Comment:
???

whaaaaa?

Reply:
I dunno why this was being filtered out from the clang-format lint, but the lint filter doesn't apply to the precommit hook.

(probably) cc @linusg and @BenWiederhake